test_reload fails sometimes

```
======================================================================
FAIL: test_reload (blocks.queue_block.tests.test_queue_block.TestQueue)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/chris/nio/projects/nio2/project/blocks/queue_block/tests/test_queue_block.py", line 159, in test_reload
    self.assert_num_signals_notified(2, blk)
  File "/Users/chris/dev/nioinnovation/nio/nio/testing/block_test_case.py", line 220, in assert_num_signals_notified
    self.assertEqual(num, signals)
AssertionError: 2 != 4
```

--- a/tests/test_queue_block.py
+++ b/tests/test_queue_block.py
@@ -153,10 +153,10 @@ class TestQueue(NIOBlockTestCase):
         blk.process_signals(signals)
         self.assertEqual(len(blk._queues['cherry']), 1)
         self.assertEqual(len(blk._queues['apple']), 1)
-        e1.wait(2)
+        e1.wait(2.5)
         self.assertEqual(len(blk._queues['cherry']), 1)
         self.assertEqual(len(blk._queues['apple']), 1)
-        self.assert_num_signals_notified(2, blk)
+        self.assert_num_signals_notified(4, blk)
         blk.stop()
 
     def test_unique(self):

